nixos/modules/config/users-groups.nix

```diff
@@ -570,6 +570,12 @@ in {
         -w ${./update-users-groups.pl} ${spec}
       '';
 
+    system.dryActivationScripts.users = stringAfter [ ]
```

Comment:
I think this is a useful feature but could this be implemented without introducing another option?
`system.activationScripts` is a module, where we could add a `.dryActivate` option or so.
Than I would set an environment variable instead of a flag to mark an activation script as `dry-able`.
Does this make sense? I think this way we would duplicate less code.

Reply:
This idea sounds better than mine, thank you for the suggestion. Going to force-push soon-ish